fix security config and some unit test

LaaSofiaa opened the pull request.

LaaSofiaa committed:
fix security config and some unit test

Samuelfdm reviewed (changes requested):
Debes devolver transaction-deploy.yml a su version original, puesto que los cambios realizados no permiten realizar despliegues

LaaSofiaa committed:
Fix environment variable syntax in workflow

LaaSofiaa committed:
Fix environment variable reference in deploy workflow

LaaSofiaa committed:
Remove refactor details for version 0.0.2 from README

Se eliminó la sección de refactor y actualizaciones de la versión 0.0.2 del README, que detallaba observaciones y cambios implementados para mejorar seguridad, mantenibilidad y eficiencia.

lilP0x committed:
se agrego jacoco para cobertura de pruebas
